### ISSUE TITLE
Read only boolean not working

### ISSUE THREAD

**Issue by infinitepr0** [glideapps/glide-data-grid#268]
```
// getCellContent()
return {
  kind: GridCellKind.Boolean,
  data: false,
  allowEdit: false,
  allowOverlay: false,
}
```

```
// onCellEdited()
([row, col], value) => {
  console.log("SHOULD NOT HAPPEN!")
}
```

How do I make a readonly Boolean cell?

**Comment by infinitepr0** [comment]
Also, any reason why its called `allowEdit` and not `readonly` like all other cells?

**Comment by jassmith** [comment]
Unfortunately it just predates the read only flag. Let me see if I can fix both these issues.

**Comment by infinitepr0** [comment]
I can help.
Shall I add an `readonly` flag, and mark `allowEdit` as depreciated?

**Comment by jassmith** [comment]
Yes but both flags must work. If both are set readonly takes precedence.

**Comment by jassmith** [comment]
Fixed in 4.0.3, it would be nice to get a PR up for readonly but at least the behavioral issue is fixed.
